doc/README.ConfigurationFiles.md

```diff
@@ -0,0 +1,66 @@
+# GEOSldas Configuration Files
+
+This document describes the files involved in the specification and pre-processing of the GEOSldas configuration parameters. 
+
+Note that the values of the configuration parameters that are used during the execution of `GEOSldas.x` are written into the GEOSldas log file (`./output/*/rc_out/Y*/M*/*.ldas_log.*.txt`).
+
+---
+### `"exeinp"` and `"batinp"` files
+
+Inputs to `ldas_setup` that contain user-defined configuration information.  
+
+Use `ldas_setup` to create sample files that contain descriptions of  the configuration parameters.  
+
+For details, see [README.md](https://github.com/GEOS-ESM/GEOSldas/blob/main/README.md).
+
+
+---
+### `LDASsa_SPECIAL_inputs_*.nml`
+ 
+Optional Fortran namelist (nml) files that contain additional user-defined configuration information for ensemble perturbations and data assimilation.  
+
+The path to these nml files is specified in the `"exeinp"` file. 
+
+During `ldas_setup`, **default** configuration files (`LDASsa_DEFAULT_inputs_*.nml`) are created in the experiment `./run` directory.  These files contain a complete set of the available configuration parameters with descriptions.  The default configuration is a single-member land model simulation without perturbations and without data assimilation. The "DEFAULT" nml files must be present in the experiment `./run` directory and should not be edited.  
+
+To run GEOSldas with ensemble perturbations and data assimilation, users must create "SPECIAL" nml files (`LDASsa_SPECIAL_inputs_*.nml`) that contain the desired settings of the parameters.  Only the nml parameters that are different from those in the "DEFAULT" files need to be included in the "SPECIAL" nml files.
+
+Parameters are grouped into three separate nml files:
+* `ensprop` : Perturbations applied during the land model ensemble propagation step. 
+* `ensupd`  : Assimilated observations and parameters of the ensemble-based analysis.
+* `catbias` : Dynamic bias estimation (defunct).
+
+
+---
+### `CAP.rc` and `cap_restart`
+
+Created by `ldas_setup`.
+
+`CAP.rc` contains the experiment start/end times and the number/length of job segments. See documentation in `"exeinp"` file.
+
+`cap_restart` contains the start time of the next job segment.  `GEOSldas.x` reads the start time from this file. [_**CORRECT???**_]
```

Comment:
@weiyuan-jiang : Is the 2nd sentence in this line correct?

Reply:
Yes, it is correct. The start time is from the file "cap_restart"